style.css

```diff
@@ -239,7 +240,7 @@
   align-items: center;
   padding: 12px;
   gap: 10px;
-  width: 124px;
+  width: 33.067%;
   height: 48px;
   background: #ff6b00;
   flex: none;
```

Comment:
- [ ] I think it's better not to use `flex` for this button that is causing alignment problems. Your portfolio will look more professional.

Reply:
done